bin/stacks/routing-dashboard-stack.ts

```diff
@@ -440,6 +437,27 @@ export class RoutingDashboardStack extends cdk.NestedStack {
               stat: 'Average',
             },
           },
+          {
+            type: 'metric',
+            width: 24,
+            height: 9,
+            properties: {
+              view: 'timeSeries',
+              stacked: false,
+              metrics: [
+                ...poolCacheLambdaMetrics,
+                ...(ipfsPoolCacheLambdaName
+                  ? [
+                    ['AWS/Lambda', 'Errors', 'FunctionName', ipfsPoolCacheLambdaName],
+                    ['.', 'Invocations', '.', '.'],
+                  ]
+                  : []),
+              ],
+              region: region,
+              title: 'Pool Cache Lambda Error/Invocations | 5min',
+              stat: 'Sum',
+            },
+          },
```

Comment:
This corresponds to 429?

Reply:
nvm, I just saw this is existing, with the positioning changes.

Reply:
I just moved that from the top to the bottom actually, I'm not sure what it means though